website/pages/en/versions.js

```diff
@@ -15,6 +15,45 @@ const CWD = process.cwd();
 const siteConfig = require(CWD + '/siteConfig.js');
 const versions = require(CWD + '/versions.json');
 
+class VersionItem extends React.Component {
+  render() {
+    const version = this.props.version;
+    const versionName = version === 'next' ? 'Master' : version;
```

Comment:
Did you use the "Master" name for simplicity or is it a typo ? 😄

Reply:
Yea since they’re not sssociated to any version you need to explicitly set a name.

Reply:
I meant "Master" instead of "Next", my bad*

Reply:
Lol what should we name it? I just took the pattern from the react-native docs

Reply:
@iRoachie I think it's better to name it `next` for consistency sake.

## Header version name
<img width="299" alt="screen shot 2018-03-07 at 12 13 13 am" src="https://user-images.githubusercontent.com/7840686/37080915-a3862ffe-219c-11e8-8448-7976e8519b34.png">

##Display version name
<img width="335" alt="screen shot 2018-03-07 at 12 13 18 am" src="https://user-images.githubusercontent.com/7840686/37080916-a3a338a6-219c-11e8-8d31-a7aff72e228d.png">

This is kinda confusing. Thoughts?

Reply:
Changed in https://github.com/react-native-training/react-native-elements/pull/915/commits/f319ad3f2a93d7f2ff9107d0027acfbdaf39e04f